vss-extension.json

```diff
@@ -88,6 +88,21 @@
 							"isRequired": true,
 							"dataType": "string"
 						}
+					},
+					{
+						"id": "gitlaburl",
+						"name": "GitLab Url",
+						"description": "git lab url of your gitlab server",
+						"inputMode": "textbox",
+						"isConfidential": false,
+						"validation": {
+							"isRequired": true,
+							"dataType": "string"
+						},
+						"values": {
+							"defaultValue": "https://gitlab.com",
+							"inputId": "gitlaburlValues"
+						}
```

Comment:
@matthieupetite Thanks for the PR. I am working on this now and I don't think we need a new field to hold this URL. The existing textbox which holds the "Server URL" can be used to connect to gitlab.com or on-premises server URL.
  
![image](https://user-images.githubusercontent.com/2685029/53157955-7a169400-35ba-11e9-89f1-da51fc1b0b16.png)

Reply:
Ok but I use also an In House Team Foundation Server which didn't display that dialog while editing the credential ?

Reply:
![image](https://user-images.githubusercontent.com/4076681/53161030-73434d80-35c9-11e9-8548-974fae84ce7d.png)

here is the result on my on prem Team Foundation Server

Reply:
> Ok but I use also an In House Team Foundation Server which didn't display that dialog while editing the credential ?

That is really odd. I just tested on latest Azure DevOps server and I see the textbox. What is your TFS version?

![image](https://user-images.githubusercontent.com/2685029/53168836-80653a00-35d3-11e9-98bc-6251e5db6ba4.png)